src/e-start.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+const childProcess = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const semver = require('semver');
+const program = require('commander');
+
+const evmConfig = require('./evm-config');
+const { color, fatal } = require('./utils/logging');
+
+program
+  .allowUnknownOption()
+  .option(
+    '--v <version>',
+    'A specific version binary of Electron to run the specified app with. Must have set the binary directory in the current config.',
+  )
+  .option('--path <path>', 'Path to an Electron executable')
+  .on('--help', () => {
+    console.log('');
+    console.log('Examples:');
+    console.log('');
+    console.log('  $ e start .');
+    console.log('  $ e start /path/to/app');
+    console.log('  $ e start --v=1.2.3 /path/to/app');
+    console.log('  $ e start /path/to/app --js-flags');
+  })
+  .parse(process.argv);
+
+try {
+  const config = evmConfig.current();
+
+  let exec;
+  let args;
+  if (program.v) {
+    const binDir = config.electronBinaryDirectory;
+    if (!binDir || !fs.existsSync(binDir)) {
+      console.error(`${color.err} electronBinaryDirectory not defined in config or not valid`);
```

Comment:
We probably don't want to do this since we need to tell than exact _what_ is not defined - if `electronBinaryDirectory` this will log `undefined` which doesn't tell them what to fix

Reply:
Oo you're right. I meant to type `'electronBinaryDirectory'` rather than `electronBinaryDirectory`